src/WP.php

```diff
@@ -16,7 +16,7 @@
  *
  * @package ee-cli
  */
-class Site_WP_Command extends EE_Site_Command {
+class WP extends EE_Site_Command {
```

Comment:
Now that class name is not long we can use WordPress as the class name. Please update it as WordPress and rename file to wordpress.php.